proposals/0082-relax-transaction-constraints.md

```diff
@@ -0,0 +1,173 @@
+---
+simd: '0082'
+title: Relax Transaction Constraints
+authors:
+  - Andrew Fitzgerald (Solana Labs)
+category: Standard
+type: Core
+status: Draft
+created: 2023-10-30
+feature:
+---
+
+## Summary
+
+This proposal aims to relax some of the constraints on which individual
+transactions can be included in a valid block.
+The proposal does not relax the constraints required for a transaction to be
+executed.
+
+## Motivation
+
+The current protocol places many constraints on the structure and contents
+of blocks; if any constraints are broken, block-validators will mark the block
+as invalid.
+Many of these constraints are necessary, but some of them are not, and lead to
+additional complexity in the protocol.
+This proposal aims to relax some of the constraints at the individual
+transaction level, in order to simplify the protocol, and give more flexibility
+to block-producer and block-validator implementations.
+
+More specifically, this proposal aims to relax the constraints which require
+account state in order to determine the validity of a transaction's inclusion.
+The reason these constraints are targetted specifically, is that reliance on
+account state necessarily requires synchronous execution within the protocol.
+This proposal on its' own, will not enable asynchronous execution, but it will
+remove one of the barriers to asynchronous execution.
+
+## Alternatives Considered
+
+1. Do nothing
+    - This is the simplest option, as we could leave the protocol as is.
+    However, this leaves the protocol more complex than it needs to be.
+2. Relax all but fee-paying constraint
+    - This was actually the initial proposal, but it was decided that it would
+    be better to relax as many constraints as possible, rather than just some.
+    Any reliance on account state, means the protocol requires synchronous
+    execution.
```

Comment:
To clarify, "it would be better to relax as many constraints as possible" just means that the fee-paying constraint is _also_ removed right? I think you could say that directly here.